iommi/form.py

```diff
@@ -827,6 +831,8 @@ def choice_to_options_selected(self):
 
     @property
     def choice_tuples(self):
+        # Why does this cache exist?  choice_tuples is only called once to render
```

Comment:
Does anybody have an answer to this?

Reply:
I think you're right. It was probably useful at some point in the past and wasn't removed when it wasn't needed anymore...